exporter/loggingexporter/logging_exporter.go

```diff
@@ -278,11 +278,26 @@ func attributeValueToString(av pdata.AttributeValue) string {
 		return strconv.FormatFloat(av.DoubleVal(), 'f', -1, 64)
 	case pdata.AttributeValueINT:
 		return strconv.FormatInt(av.IntVal(), 10)
+	case pdata.AttributeValueARRAY:
+		return attributeValueArrayToString(av.ArrayVal())
 	default:
 		return fmt.Sprintf("<Unknown OpenTelemetry attribute value type %q>", av.Type())
 	}
 }
 
+func attributeValueArrayToString(av pdata.AnyValueArray) string {
+	retStr := "["
```

Comment:
While likely not a real performance issue, any reason not to use https://golang.org/pkg/strings/#Builder?

Reply:
The reason would be my inexperience with golang :) will change to use the builder.